scripts/log-viewer.sh

```
@@ -0,0 +1,97 @@
+CONTEXT=2
+FILTER=""
+
+# Parse args
+while [ $# -gt 0 ]; do
+    case "$1" in
+        --context)
+            CONTEXT="$2"
+            shift 2
+            ;;
+        --context=*)
+            CONTEXT="${1#*=}"
```

Shellcheck (0.11.0)

[warning] CONTEXT appears unused. Verify use (or export if used externally).

(SC2034)